README.md

```diff
@@ -74,6 +92,16 @@ optional arguments:
                         created and saved to that filename.
   --image_path IMAGE_PATH
                         search path for FITS postage stamp images.This will be
-                        prepended to any existing IMSIM_IMAGE_PATHenvironment
-                        variable, for which $CWD is included bydefault.
+                        prepended to any existing IMSIM_IMAGE_PATH environment
+                        variable, for which $CWD is included by default.
 ```
+
+## Release Status
+The list of released versions of this package can be found [here](https://github.com/LSSTDESC/imSim/releases), with the master branch
+including the most recent (non-released) development.  imSim is currently
+still considered beta code.
```

Comment:
This paragraph could use some clarifying, I think.  At least in the repo guidelines, we had suggested referring to releases as versions that are intended for general use.  If you're saying it's beta code, then it seems like those are tagged versions but not releases?  If I follow the link, then you appear to have tagged versions and pre-releases (which I think of as another name for a tagged version of beta code) but no releases at all.  So one possible rewording might be "imSim is currently considered beta code until its v1 release, which will be accompanied by a paper that should be cited by those who use the released version in published work.  A list of tagged versions and pre-releases of this package can be found [here](https://github.com/LSSTDESC/imSim/releases), with the master branch including the most recent development beyond the latest pre-release."  - assuming that is indeed what you mean here.

Reply:
I've made the requested changes here and above.  

For the above point there is a really confusing thing that GH does that is good to appreciate I thought I would write down for people.  You can see their explanation of it here:

https://github.community/t5/How-to-use-Git-and-GitHub/Tag-without-release/td-p/6255

Basically releases happen when you add release notes.  And if you use the API you can get them.  They show up on the "release tab".  *But*, all code tags show up there too.  So if you tag the code without a release it shows up on that page. It has a sort of different style.  At the very bottom of the imSim release page you can see a tag.  The others actually are releases made by Jim but he checked a box "pre-release" when he did it to let people know it is still non-production code.

I made it clearer that this is still considered pre-release code.